Install-Guide/Community_Packages.md

```diff
@@ -1,35 +1,38 @@
 ### Community Packages
 
-An **X** means packages are planned to be in the respective repository.  
-A **—** means we have no plans to build the version for the repository.  
-**d.g.o** means packages will (also) be provided on https://download.gluster.org  
-**DNF** means the packages are installed from the Fedora Updates or Updates-testing repos.  
+An **X** means packages are provided in the respective repository.  
```

Comment:
It wasn't ever a list.
And it looks correct pasted into, e.g., http://dillinger.io/

???